spec/design/skel/X64/FPU_H.thy

```diff
@@ -13,7 +13,9 @@ imports
 begin
 context Arch begin arch_global_naming (H)
 
-#INCLUDE_HASKELL SEL4/Object/FPU/X64.hs CONTEXT X64_H ArchInv=Arch
+#INCLUDE_HASKELL SEL4/Object/FPU/AARCH64.hs CONTEXT AARCH64_H decls_only ArchInv=Arch ONLY fpuOwner_asrt
```

Comment:
wait what? this in an X64 file

Reply:
Whoops, copy paste error, thanks for spotting that. It's mildly surprising that that doesn't break anything.